lib/gui/app/components/settings/index.ts

```diff
@@ -1,5 +1,13 @@
 /*
+<<<<<<< HEAD
  * Copyright 2019 balena.io
+=======
```

Comment:
this looks broken

Reply:
Didn't fire a conflict since it was in a comment probably. Fixing